spark_pipeline_framework/transformers/http_data_receiver/v4/http_data_receiver.py

```diff
@@ -0,0 +1,277 @@
+import json
+from typing import Any, Dict, List, Optional, cast
+
+from more_itertools import chunked
```

Comment:
since we're only using more_itertools for this. We could have created a util for this instead of installing a new pip.